servant-util/src/Servant/Util/Combinators/Logging.hs

```diff
@@ -384,6 +384,16 @@ instance ( HasServer (Verb mt st ct a) ctx
         inRouteServer @(Verb mt st ct a) route $
         applyLoggingToHandler (Proxy @config) (Proxy @mt)
 
+instance ( HasServer (Verb mt) ctx
```

Comment:
Probably something else should be in place of `HasServer (Verb mt) ctx`.

Reply:
:fffuuu: